scipy/stats/tests/mpmath_distribution.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
+from mpmath import mp
+
+mp.dps = 100  # default in case the user forgets to set it
+
+class Distribution:
```

Comment:
What should we call this? `ReferenceDistribution`?

Reply:
This, or `MPMathDistribution`.